packages/@ourworldindata/grapher/src/entitySelector/EntitySelector.tsx

```diff
@@ -60,162 +56,377 @@ export class EntitySelector extends React.Component<{
+                            checked={this.isEntitySelected(name)}
+                            onChange={() => this.onChange(name)}
+                        />
+                    </li>
+                ))}
+            </ul>
+        )
+    }
+
+    private isEntitySelected(name: string): boolean {
+        return this.selectionArray.selectedSet.has(name)
+    }
+
+    private renderAllEntitiesInMultiMode(): JSX.Element {
+        const { selected, unselected } = this.partitionedAvailableEntities
 
         return (
-            <div
-                className={classnames(
-                    "EntitySelector",
-                    this.isMulti
-                        ? "EntitySelectorMulti"
-                        : "EntitySelectorSingle"
-                )}
+            <Flipper
+                spring={{
+                    stiffness: 300,
+                    damping: 33,
+                }}
+                flipKey={this.selectionArray.selectedEntityNames.join(",")}
             >
-                <div className="searchBar">
-                    <div className="searchInput">
-                        <input
-                            type="search"
-                            placeholder="Search..."
-                            value={searchInput}
-                            onChange={(e): void => {
-                                this.searchInput = e.currentTarget.value
-                            }}
-                            onKeyDown={this.onSearchKeyDown}
-                            ref={(e): HTMLInputElement =>
-                                (this.searchField = e as HTMLInputElement)
-                            }
-                        />
-                        <FontAwesomeIcon icon={faMagnifyingGlass} />
-                    </div>
-                    {this.isMulti &&
-                    selectionArray.selectedEntityNames.length > 0 ? (
+                {selected.length > 0 && (
+                    <Flipped flipId="__selection" translate opacity>
+                        <div className="section-title grapher_body-3-medium-italic">
+                            Selection
+                        </div>
+                    </Flipped>
+                )}
+                <ul>
+                    {selected.map((name) => (
+                        <Flipped key={name} flipId={name} translate opacity>
+                            <li
+                                className={cx("animated-entity", {
+                                    "most-recently-selected":
+                                        this.mostRecentlySelectedEntityName ===
+                                        name,
+                                })}
+                            >
+                                <SelectableEntity
+                                    name={name}
+                                    type="checkbox"
+                                    checked={this.isEntitySelected(name)}
```

Comment:
Can replace this with `true` in theory; we already know that this entity is selected (same below for `unselected`).